src/pysight/nd_hist_generator/frame_chunk.py

```diff
@@ -180,50 +183,54 @@ def __linspace_along_sine(self) -> np.ndarray:
 
         return np.array(pts)
 
+
     def _hist_with_flim(
-        self, data: List[np.ndarray], edges: List[np.ndarray], chan: int
+        self, flim_edges: List[np.ndarray], edges: List[np.ndarray], chan: int
```

Comment:
Why not directly receive a data object, and at the caller pass `self.df_dict[chan].copy(deep=True)`?

Reply:
Fixed da5c503